### DISCUSSION TITLE
Rotation Convention when Specifying Orientation

### DISCUSSION THREAD

**Discussion by mvailhe** [neperfepx/neper#887] [Q&A] [answered]
I am using the capabilities of Neper as shown in the tutorial to specify orientation information for tessellation cells as shown below.

`neper -T -n 1000 -ori "odf(mesh=file(fr.msh),val=file(odf.txt))" -crysym cubic`

The fundamental region files I am using are created in a different program that uses passive convention. I saw in the Neper documentation that "the convention can be added to the descriptor, either active or passive, as in rodrigues:active. When no convention is provided, active is assumed." Is there a way to incorporate into the code above a way to specify that the odf has a passive convention?

If not, is there a way that upon exporting the .tess or .msh files created in Neper one can change the convention to stay consistent with other programs like MechMet?

**Comment by mkasemer** [comment]
@rquey : This is a discussion that was ongoing with folks from Tresa Pollock's group (@mvailhe and @mechlin), along with @dplab. I think this is a good point, in what convention is the ODF defined and can we have control over it. I think there are also questions we need to address once and for all re: #641.

If it makes sense to take it private to -dev, please do so.

**Comment by rquey** [comment]
@mvailhe The short answer is that I had the convention names switched in Neper. So, what is generally understood to be the "passive" convention is called the "active" convention in Neper, and vice versa. It has no special implications - it's just a terminology issue, but I'm working on a fix.

The passive convention will become the new default, and the ODF space will also be understood to be in passive convention (as it already is). I may add an option to define the convention, but this should take care of your concerns.

**Reply by rquey** [reply, marked as answer]
@mvailhe Version 4.10.0 includes the orientation convention fix, now "passive" by default.
